### ISSUE TITLE
Coordinates should be checked against area directive coverage

### ISSUE THREAD

**Issue by marangonico** [nofaceinbook/muxp#25]
Usage of coordinates out of boundary defined by area directive, could lead to undesired effects.

Probably this kind of scenario, could raise a simple warning.

**Comment by nofaceinbook** [comment]
Yes. Good point. Could also define area by reading coordinates, so area could be optional.
